src/main/resources/db/migration/common/V1_403__manual_deletion_audit_activity.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO audit_activity (aua_id, activity_name, activity_description, created_ts, created_by, last_modified_ts, last_modified_by)
+VALUES (36, 'Manual Deletion', 'Manual Deletion', CURRENT_TIMESTAMP, 0, CURRENT_TIMESTAMP, 0);
```

Comment:
Duplicate of https://github.com/hmcts/darts-api/pull/2133/files#diff-0599014aa9b7da6978e32927fdd55ef52e3b23eb55ce641dcede54a50db981cc Ensure only merged once.